net.mkiol.Jupii.yaml

```
@@ -0,0 +1,333 @@
+app-id: net.mkiol.Jupii
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+rename-icon: jupii
+command: jupii
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all # v4l2 webcams
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-videos
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.Tracker1.Steroids # media files index
+  - --talk-name=org.freedesktop.Tracker3.Endpoint # media files index
+  - --talk-name=org.kde.StatusNotifierWatcher # system tray icon
+  - --env=QT_QUICK_CONTROLS_STYLE=org.kde.desktop
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - "*/cmake/*"
+  - "*.a"
+  - "*.la"
+modules:
+  - name: taglib
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      - -DBUILD_TESTING=OFF
+      - -DBUILD_BINDINGS=OFF
+      - -DBUILD_SHARED_LIBS=ON
+      - -DENABLE_STATIC_RUNTIME=OFF
+    cleanup:
+      - /bin
+      - /share
+      - /include
+      - /lib/cmake
+      - /lib/pkgconfig
+      - "*.a"
+      - "*.la"
+    sources:
+      - type: archive
+        url: https://taglib.org/releases/taglib-1.13.1.tar.gz
+        sha256: c8da2b10f1bfec2cd7dbfcd33f4a2338db0765d851a50583d410bacf055cfd0b
+
+  - name: gumbo
+    buildsystem: autotools
+    config-opts:
+      - --with-pic
+      - --enable-shared
+      - --disable-static
```

```suggestion
    config-opts:
      - --disable-static
```

I don't think you need any of these.